generator/src/googleapis/codegen/api.py

```diff
@@ -330,6 +330,10 @@ def _NormalizeUrlComponents(self):
       self._api.SetTemplateValue('rootUrl', '%s://%s/' % (scheme, service_host))
     if service_path is None:
       self._api.SetTemplateValue('servicePath', base_path[1:])
+    
+    if not self.values.get('mtlsRootUrl') and root_url:
+      mtls_root_url = root_url.replace('googleapis.com', 'mtls.googleapis.com')
+      self._api.SetTemplateValue('mtlsRootUrl', mtls_root_url)
```

Comment:
This is temporary until MTLS is rolled out to all services, correct? If so, let's add a comment/TODO to remove it when that happens

Reply:
done